pyproject.toml

```diff
@@ -36,6 +36,7 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
```

Comment:
Do you want to list this in beta-support already or should we wait until 3.13 has an official release?

Reply:
why not, if we are fully python 3.13 compatible, we can list it immedieately no?